### PR TITLE
Fix model_class_name attribute

### PR DESCRIPTION
**Pull request by Dynnammo**
Found a bug with @armandfardeau when using `decidim-calendar` in a 0.21 instance of Decidim : an attribute `model_class_name` required in version 0.2x isn't defined in current version of this module

I try to correct it as it seems to be the only one that triggers important issues.

EDIT : Also got a problem when wanting to display the Gantt diagram. I'll fix it in another PR
EDIT: Also got a problem when integrating `decidim-module-term_customizer`. As it too concerns the `model_class_name` fix, should I do another PR that also should be merged after this one @mijailr ?

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -42,3 +42,8 @@ fr:
         calendar: Calendrier
         gantt: Graphe de Gantt
         ical: Télécharger iCAL
+  activerecord:
```
Comment (mijailr) [inline config/locales/fr.yml:45]: Maybe you can add the translations on crowdin. https://crowdin.com/project/decidim-calendar All the translation are managed there.

Comment (Dynnammo) [reply config/locales/fr.yml:45]: :ok_hand:

**Comment by Dynnammo** [comment]
I also tried to write a test to verify that the help written in help sections in correctly displayed when accessing the Calendar component, however I'm struggling to correctly fill the form...

I tried this, in the `help_section.rb` : 

```ruby

  it "displays correctly what was specified" do
    fill_in_i18n(
      :help_sections_sections_calendar_content,
      "#help_sections_sections_calendar_content-tabs",
      en: 'Help for calendar',
      es: "Ayuda de calendario",
      ca: "Ajuda del calendari"
    )
    page.find("button[name='commit']").click
    expect(page).to have_content("successfully")
  end
```

Error gotten : `Capybara::ElementNotFound: Unable to find css "#help_sections_sections_calendar_content-tabs"`

**Comment by mijailr** [comment]
LGTM, @andreslucena or @paranoix can we merge this?

**Comment by andreslucena** [comment]
Hey @mijailr I'm not working with aLabs anymore, I don't have permission to merge this. @iokese or @paranoix can do it.

**Pull request merged**